src/codeu/chat/server/Model.java

```diff
@@ -68,9 +74,95 @@ public int compare(Time a, Time b) {
   private final Uuid.Generator userGenerations = new LinearUuidGenerator(null, 1, Integer.MAX_VALUE);
   private Uuid currentUserGeneration = userGenerations.make();
 
+  private final String database = "codeutest.db";
+  private final String userTable = "USER";
+  private final String sqliteClass = "org.sqlite.JDBC";
+
+  public Model () {
+    Connection conn = null; // Placeholder for connection to database
+    Statement stmt = null; // Placeholder for sql command
+    try {
+
+      // Loads the JDBC class dynamically to handle the SQL connection
+      // DriverManager cannot create the connection if this class isn't loaded at runtime
+      Class.forName(sqliteClass);
+      
+      // Connect to database. Creates database if it doesn't exist.
+      conn = DriverManager.getConnection("jdbc:sqlite:" + database);
+      DatabaseMetaData dbm = conn.getMetaData();
+      ResultSet rs = dbm.getTables(null, null, userTable, null);
+       
+      if (rs.next()) { // Table exists
+       
+        String sql = "SELECT ID, TIME, NAME FROM " + userTable;
+        stmt = conn.createStatement();
+        ResultSet users = stmt.executeQuery(sql);
+        // loop through the result set
+        while (users.next()) {
+          
+          String username = users.getString("NAME");
+          String uuidString = users.getString("ID");
+          uuidString = uuidString.substring(6, uuidString.length() - 1);
```

Comment:
Why are we getting a subsctring starting at the 6th  index and excluding the last index? Can you add a comment describing why this is the case? Also, might be a good idea to define a named constant for what the value 6 represents:
https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants